src/client/pages/AssessmentSection/Descriptions/GeneralComments/GeneralComments.tsx

```diff
@@ -4,20 +4,22 @@ import { useTranslation } from 'react-i18next'
 import CommentableDescription from '../CommentableDescription'
 
 type Props = {
+  assessmentName: string
   sectionName: string
   disabled: boolean
 }
 
 const GeneralComments: React.FC<Props> = (props) => {
-  const { sectionName, disabled } = props
+  const { assessmentName, sectionName, disabled } = props
+  const isPanEuropean = assessmentName === 'panEuropean'
 
   const { i18n } = useTranslation()
```

Comment:
Better to use `const { t } = useTranslation` and usage `t('translation.key')` to avoid `<string>`
Just minor code improvement even if this didn't change in your PR but we can update these when we edit files where it's still using i18n